servers/zms/src/test/resources/solution_templates.json

```diff
@@ -236,6 +236,97 @@
                 ]
             }
         ]
+    },
+    "templateWithService" : {
+        "roles": [
+            {
+                "name": "_domain_:role.vip_admin",
+                "modified": "1970-01-01T00:00:00.000Z"
+            }
+        ],
+        "policies": [
+            {
+                "name": "_domain_:policy.vip_admin",
+                "modified": "1970-01-01T00:00:00.000Z",
+                "assertions": [
+                    {
+                        "resource": "_domain_:vip*",
+                        "role": "_domain_:role.vip_admin",
+                        "action": "*"
+                    }
+                ]
+            }
+        ],
+        "services": [
+            {
+                "name": "testService",
```

Comment:
The correct value here must be: "name": "_domain_:testservice". Please update all other test cases and handling of the service name accordingly.